src/commands/variable.command.ts

```diff
@@ -0,0 +1,12 @@
+import {variableService} from "../services/package-manager/variable-service";
+
+export class VariableCommand {
+
+    public async listAssignments(variableType: string, jsonResponse: boolean, params: string): Promise<void> {
+        if (jsonResponse) {
+            await variableService.listCandidateAssignments(variableType, params);
+        } else {
+            await variableService.findAndExportCandidateAssignments(variableType, params);
+        }
```

Comment:
The nested statements should be the other way around, as it should export when `--json` flag is passed.